### PR TITLE
ci: Updating pr-windows-signed-scripts.yaml to allow for overriding cluster definition

### PR DESCRIPTION
**Pull request by marosset**
…

Default value is 
examples/e2e-tests/kubernetes/windows/hybrid/definition.json

**Reason for Change**:

This will help us test updates to the windows provisioning scripts for even more use cases

**Issue Fixed**:

**Credit Where Due**:

Does this change contain code from or inspired by another project?

- [x] No
- [ ] Yes

If "Yes," did you notify that project's maintainers and provide attribution?

- [ ] No
- [ ] Yes

**Requirements**:

- [x] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [x] includes documentation
- [ ] adds unit tests
- [ ] tested upgrade from previous version

**Notes**:

### CODE REVIEW

File: .pipelines/pr-windows-signed-scripts.yaml [hunk 1]
```diff
@@ -46,7 +46,7 @@ jobs:
       -e CLEANUP_ON_EXIT="true" \
       -e CLIENT_ID=${CLIENT_ID} \
       -e CLIENT_SECRET="$(CLIENT_SECRET)" \
-      -e CLUSTER_DEFINITION="examples/e2e-tests/kubernetes/windows/hybrid/definition.json" \
+      -e CLUSTER_DEFINITION="${CLUSTER_DEFINITION}" \
```
Comment (jsturtevant) [inline .pipelines/pr-windows-signed-scripts.yaml:49]: Where is this defaulted? Maintainers would have to trigger the pipeline manually and set this value?

Comment (marosset) [reply .pipelines/pr-windows-signed-scripts.yaml:49]: It is defaulted in the AzureDevops pipeline job definition.
Yes, maintainers would need to trigger this manually and override the variable.

**Comment by jsturtevant** [comment]
can you add a note to https://github.com/Azure/aks-engine/blob/master/docs/topics/windows-provisioning-scripts.md about how these are tested?

File: docs/topics/windows-provisioning-scripts.md [hunk 2]
```diff
@@ -9,12 +9,22 @@ This document details how to update and test changes to these files.
...
+
 Changes to the provisioning scripts packaged in the .zip can be tested by:
 
 - Running [scripts/build-windows-provisioning-scripts.sh](../../scripts/build-windows-provisioning-scripts.sh) which will create a .zip and upload it to a storage account.
 - Create/update a cluster definition file and set `windowsProfile.provisioningScriptsPackageURL' to point to the uploaded back.
 - Perform a cluster deployment with aks-engine.
 
+### CI Pipeline validation
+
+Changes to the privisioing scripts are also tested with an Azure DevOps pipeline job enabled in Azure/aks-engine.
```
Comment (jsturtevant) [inline docs/topics/windows-provisioning-scripts.md:22]:
```suggestion
Changes to the provisioning scripts are also tested with an Azure DevOps pipeline job enabled in Azure/aks-engine.
```

**Pull request merged**